ocs_ci/utility/svt.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shlex
+from subprocess import Popen, PIPE
+from ocs_ci.framework.testlib import workloads
+import logging
+import shutil
+import pytest
+from ocs_ci.utility.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def svt_project_clone():
+    run_cmd("git clone https://github.com/openshift/svt.git /tmp/svt")
+
+
+@pytest.fixture(scope='function')
+def svt_create_venv_setup():
+    run_cmd("virtualenv -p /bin/python2 /tmp/venv")
+    run_cmd("/bin/sh -c 'source /tmp/venv/bin/activate && python --version'")
+    run_cmd("/bin/sh -c 'source /tmp/venv/bin/activate && pip install -r registry_requirement.txt'")
+
+
+@pytest.fixture(scope='function')
+def svt_cluster_loader(clusterload_file="/tmp/svt/openshift_scalability/config/master-vert.yaml"):
+    KUBECONFIG = os.environ["KUBECONFIG"]
+    """
+    https://github.com/openshift/svt/blob/master/openshift_scalability/README.md
+
+    Arguments:
+        -f : This is the input config file used to define the test.
+        -kubeconfig : kebeconfig path
```

Comment:
'KUBECONFIG = os.environ["KUBECONFIG"]' is same as "os.getenv('KUBECONFIG')" ?

Reply:
Yes, updated with os.getenv as its a better solution